core/scheduler/service/executor_input_compiler.go

```diff
@@ -62,6 +63,33 @@ type InputCompiler struct {
 	logger log.Logger
 }
 
+func sanitiseLabel(key string) string {
+	// implements validation from https://cloud.google.com/bigquery/docs/labels-intro#requirements
+
+	// max length 63 chars
+	// if length greater than 63 then truncate to 61 and add __ at end to denote truncation
+	if maxLabelLength := 63; len(key) > maxLabelLength {
+		key = key[:61] + "__"
```

Comment:
this was supposed to be other way, right ?  -- `key[len(key)-61:]`

Reply:
done

Reply:
since we're truncating from last, I believe it should be look like this `"__" + key[len(key)-61:]`

Reply:
correct